data/Kazakhstan/Assembly/sources/instructions.json

```diff
@@ -23,6 +23,33 @@
       }
     },
     {
+      "file": "morph/term-6.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/kazakhstan-mazhilis",
+        "query": "SELECT * FROM data ORDER BY id"
+      },
+      "source": "http://www.parlam.kz",
+      "merge": {
+          "incoming_field": "name",
+          "existing_field": "name",
+          "reconciliation_file": "reconciliation/term-6.csv"
+      },
+      "type": "membership"
+    },
+    {
+      "file": "manual/terms.csv",
+      "type": "term"
+    },
+    {
+      "file": "gender-balance/results.csv",
+      "type": "gender",
+      "create": {
+        "from": "gender-balance",
+        "source": "Kazakhstan/Assembly"
+      }
+    },
```

Comment:
You seem to have doubled up the import of the terms file and the gender-balance file here…